pkg/common/config/option.go

```diff
@@ -151,6 +152,10 @@ func NewOptions(opts []Option) *Options {
 		// like they are normal requests
 		DisablePreParseMultipartForm: false,
 
+		// Print logs of the routes by default
+		// Will not be printed when set to False
+		DebugPrintRoute: true,
```

Comment:
- DebugPrintRoute -> DisablePrintRoute
- DisablePrintRoute  default  value false and print log

Reply:
Done